lib/models/tree.model.ts

```diff
@@ -18,9 +18,13 @@ export class TreeModel implements ITreeModel {
   eventNames = Object.keys(TREE_EVENTS);
 
   setData({ nodes, options, events }) {
-    this.options = new TreeOptions(options);
+    if (options) {
+      this.options = new TreeOptions(options);
+    }
```

Comment:
If the options property doesn't change, a null value is passed in - but we don't want to wipe out the pre-existing options property.